### PR TITLE
Fix recipe and update v2.0.2

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #5
Closes #6
Closes #7
Closes #8

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,49 +1,64 @@
...
+    - nc2csv=nappy.script.nc2csv:nc2csv
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
 
 requirements:
   build:
-    - python
-    - setuptools
   host:
```
Comment (BastianZim) [inline recipe/meta.yaml:23]: This is slightly weird because the package imports itself during building so all of `run` is also required in `host`.

Comment (ReimarBauer) [reply recipe/meta.yaml:23]: earlier versions did not have this dependency

Comment (BastianZim) [reply recipe/meta.yaml:23]: It comes from here where they import the version: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=476348&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=447

**Comment by BastianZim** [comment]
It's working now.

File: recipe/meta.yaml (hunk 1 above)
Comment (ReimarBauer) [reply recipe/meta.yaml:23]: https://github.com/cedadev/nappy/blob/main/nappy/__init__.py

**Comment by ReimarBauer** [comment]
Thx :)

**Pull request merged**